CMakeLists.txt

```diff
@@ -90,27 +90,27 @@ if( NOT fiat_FOUND )
   list(APPEND srcs ${ABOR1_PATH} ${OML_PATH} ${PARKIND1_PATH})
 endif()
 
-list(APPEND srcs field_basic_module.F90 field_init_debug_value_module.F90 dev_alloc.c)
+list(APPEND srcs field_basic_module.F90 field_init_debug_value_module.F90 dev_alloc.c field_constants_module.F90 field_abort_module.F90)
 
 ## preprocess fypp files
 foreach (SUFF IN ITEMS IM RM RB RD LM)
   string (TOLOWER ${SUFF} suff)
   foreach (RANK RANGE 2 5)
-    foreach (FUNC IN ITEMS "" _gathscat _access _util _array_util)
-      add_custom_command (OUTPUT field_${RANK}${suff}${FUNC}_module.F90
-	      COMMAND ${FYPP} -n -DRANK=${RANK} -DSUFF='${SUFF}' -m os -M ${CMAKE_CURRENT_SOURCE_DIR} -m fieldType 
```

Comment:
Were the fypp line annotations removed intentionally? (the `-n` arg)

Reply:
Yes and no. Much easier to look at real Fortran code when debugging, but we can got back to fypp line indexing if you do not like it.